packages/blockchain-wallet-v4-frontend/src/modals/model.ts

```diff
@@ -0,0 +1,15 @@
+import { LimitWithEffective, SeamlessLimits } from 'data/types'
+
+export const getEffectiveLimit = (limits: SeamlessLimits): LimitWithEffective | undefined => {
+  const { current } = limits
+  switch (true) {
+    case current?.daily?.effective:
+      return current.daily
+    case current?.monthly?.effective:
+      return current.monthly
+    case current?.yearly?.effective:
+      return current.yearly
+    default:
+      return undefined
+  }
+}
```

Comment:
this is not the correct place for this file IMO.  a model file here should be useable by most all modals.   Perhaps we create a new service folder named `custodial` and put it in there?  thoughts @blockdylanb ?

Reply:
having it as a service then means sagas could use it as well and not be importing a model from modals which makes little sense.

Reply:
I'm up to any of those, my idea/intention was to keep it on common place and re-use it in several others since we literately need it for every flow.

Reply:
I think a service file makes sense in this case. It would be nice to have fewer model files spread throughout the code. I'd personally prefer to have a small amount of model files close to the top of the file hierarchy. I think it's ok that they are bigger files because as a developer you mostly access them by clicking the "Go to Definition" option in the code editor vs scrolling around in that file looking for things.